operator/molecule/cluster-olm/prepare.yml

```diff
@@ -0,0 +1,137 @@
+---
+
+- name: Prepare operator resources
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: '{{ ansible_playbook_python }}'
+    deploy_dir: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/deploy"
+  tasks:
+  # need to pull all the info together into a config map
+  - name: Ensure specified namespace is present
+    k8s:
+      api_version: v1
+      kind: Namespace
+      name: '{{ namespace }}'
+      
+  - name: Set Operator Image from Format
+    shell: "echo $IMAGE_FORMAT | envsubst '${component}'"
+    environment:
+      component: ansible-service-broker-operator
+    register: operator_image_raw
+    
+  - name: Set Operand Image f_om Format
+    shell: "echo $IMAGE_FORMAT | envsubst '${component}'"
+    environment:
+      component: ansible-service-broker
+    register: operand_image_raw
+  
+  - name:  Add operator image to csv
```

Comment:
We should be consistent in our names. Sometimes we write operator and above we have Operator. I'd say stick with all lowercase.